src/wallet/rpc/wallet.cpp

```diff
@@ -220,7 +220,7 @@ static RPCHelpMan loadwallet()
                 "\nNote that all wallet command-line options used when starting bitcoind will be"
                 "\napplied to the new wallet.\n",
                 {
-                    {"filename", RPCArg::Type::STR, RPCArg::Optional::NO, "The wallet directory or .dat file."},
+                    {"filename", RPCArg::Type::STR, RPCArg::Optional::NO, "The path of the wallet directory (or a legacy wallet's .dat file). Takes an absolute path or a path relative to the default wallet directory."},
```

Comment:
The relative path is relative to the wallet directory, not the default wallet directory. Starting with `-walletdir=<dir>` will change the wallet directory and specifying a wallet name in this argument will be interpreted to be relative to that walletdir.

Reply:
Thanks for the feedback - made the change to indicate that if `-walletdir` is specified, `loadwallet` will interpret argument relative to that directory.